hydra-access-controls/lib/hydra-access-controls.rb

```diff
@@ -2,6 +2,7 @@
 require 'active-fedora'
 require 'blacklight'
 require 'cancan'
+require 'action_view' # https://github.com/haml/haml/issues/695
```

Comment:
You mean here: https://github.com/projecthydra/active_fedora/blob/8.x-stable/lib/active_fedora/railtie.rb ?